agent/xds/clusters_test.go

```diff
@@ -642,6 +643,7 @@ func TestClustersFromSnapshot(t *testing.T) {
 	}
 
 	latestEnvoyVersion := proxysupport.EnvoyVersions[0]
+	latestEnvoyVersion_v2 := proxysupport.EnvoyVersionsV2[0]
```

Comment:
I realized when I went to update these golden tests that my last refactor hadn't accounted for skipping `v2-compat` tests for `>= 1.17.x`. All of the golden tests got similar adjustments.